tests/functional/fetchGit.sh

```diff
@@ -208,7 +208,7 @@ path6=$(nix eval --impure --raw --expr "(builtins.fetchTree { type = \"git\"; ur
 [[ $path3 = $path6 ]]
 [[ $(nix eval --impure --expr "(builtins.fetchTree { type = \"git\"; url = \"file://$TEST_ROOT/shallow\"; ref = \"dev\"; shallow = true; }).revCount or 123") == 123 ]]
 
-expectStderr 1 nix eval --expr 'builtins.fetchTree { type = "git"; url = "file:///foo"; }' | grepQuiet "'fetchTree' requires a locked input"
+expectStderr 1 nix eval --expr 'builtins.fetchTree { type = "git"; url = "file:///foo"; }' | grepQuiet "fetchTree requires a locked input"
```

Comment:
I wouldn't mind some sort of formatting to distinguish the identifier from regular english.

(it is semantically different e.g. if we did internationalization, we would want to make `fetchTree` as something that is not translated.)

Reply:
It's colored pink in error messages, is that enough?

Reply:
You can see where it's constructed here:
https://github.com/NixOS/nix/blob/9723f533d85133fa3c4d9421a58c7765cb61e733/src/libexpr/primops/fetchTree.cc#L179

Reply:
oh I didn't consider colors, OK.